lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/repeat-fraud-check.yaml

```diff
@@ -106,109 +106,79 @@ states:
           updateFields: family-name,address
           updateSupported: true
       dob:
-        targetState: UPDATE_NAME_DOB
-        checkFeatureFlag:
-          updateDetailsAccountDeletion:
-            targetState: UPDATE_NAME_DOB_ALLOW_ACCOUNT_DELETION
+        targetState: UPDATE_NAME_DOB_ALLOW_ACCOUNT_DELETION
```

Comment:
Yeah, it'd be nice - the only slightly annoying thing is that we'd need to do it in a two-step deployment to avoid breaking sessions (though for these screens the chance of actually hitting one is very low...)

We almost need a ticket every month to go through and revise all the names that have drifted like this(!)